DiTails/manifest.json

```diff
@@ -3,9 +3,9 @@
   "id": "DiTails",
   "name": "DiTails",
   "author": "Auros",
-  "version": "1.0.0",
+  "version": "1.1.0",
```

Comment:
Do not update the version of mods you are submitted a PR to

Reply:
fair enough, noted!